src/sentry/users/models/user.py

```diff
@@ -105,6 +105,10 @@ class User(Model, AbstractBaseUser):
     # display name
     name = models.CharField(_("name"), max_length=200, blank=True, db_column="first_name")
     email = models.EmailField(_("email address"), blank=True, max_length=200)
+    # temporary field that prevents creating new users with the same email address as an existing user on the DB level
+    email_unique = models.EmailField(
+        _("unique email address"), blank=True, max_length=200, unique=True
```

Comment:
should this be `null=True` so it's optional on the DB level (in addition to or instead of `blank=True`)